### DISCUSSION TITLE
Garbled text extraction

### DISCUSSION THREAD

**Discussion by LarrySleeper** [pymupdf/PyMuPDF#1963] [Looking for help]
Hello,

I'm having trouble extracting text from the attached PDF.

Using Python 3.6.8 and PyMuPDF==1.18.14.  

Here is my code:

import fitz
doc = fitz.open('t-s-cm-2.pdf')
for page in doc:
    print (page.get_text('text'))

The font is CCUAAA+CumberlandAMT,Regular which is TrueType.   

Is this font perhaps not supported by MuPDF?

Thank you.

Larry

[t-s-cm-2.pdf](https://github.com/pymupdf/PyMuPDF/files/9767150/t-s-cm-2.pdf)

**Comment by LarrySleeper** [comment]
Still looking for help.   Anybody?   Thank you!

**Reply by JorjMcKie** [reply]
I can extract text no problem with v1.23.20. Even extracting tables works!
